test/schemes/boundary/open_boundary/boundary_zone.jl

```diff
@@ -0,0 +1,221 @@
+@testset verbose=true "Boundary Zone" begin
+    @testset verbose=true "Boundary Zone 2D" begin
+        particle_spacing = 0.2
+        open_boundary_layers = 4
+
+        plane_points_1 = [[0.0, 0.0], [0.5, -0.5], [1.0, 0.5]]
+        plane_points_2 = [[0.0, 1.0], [0.2, 2.0], [2.3, 0.5]]
+
+        @testset "Points $(i)" for i in eachindex(plane_points_1)
```

Comment:
```suggestion
        @testset verbose=true "Points $(i)" for i in eachindex(plane_points_1)
```
In our current configuration, we set everything to verbose. But we might want to discuss that in the future.